packages/playwright-core/src/server/registry/index.ts

```diff
@@ -1069,6 +1072,13 @@ export class Registry {
       throw new Error(`ERROR: Playwright does not support ${descriptor.name} on ${hostPlatform}`);
     if (!isOfficiallySupportedPlatform)
       logPolitely(`BEWARE: your OS is not officially supported by Playwright; downloading fallback build for ${hostPlatform}.`);
+    if (descriptor.hasRevisionOverride) {
+      const message = `You are using a frozen ${descriptor.name} browser which does not receive updates anymore. Please update to the latest version of Playwright to test up-to-date browsers.`;
```

Comment:
add which OS is deprecated.

Reply:
Maybe just a link to sys requirements instead?